tests/containers/rootless_podman.pm

```diff
@@ -150,7 +152,6 @@ sub verify_userid_on_container {
     # Check for bsc#1182428
     # podman 2.1.1 with keep-id option list unexpected capabilities
     # podman of the same version can still show the nsenter original issue
-    my $podman_version = get_podman_version();
```

Comment:
Let's try to keep it here. `$podman_version = get_podman_version();`

Reply:
> Let's try to keep it here. `$podman_version = get_podman_version();`

If we do that the variable will get garbled output in s390x.